package.json

```diff
@@ -6,8 +6,17 @@
   "dependencies": {
     "fastify-plugin": "^1.0.0",
     "into-stream": "^4.0.0",
+    "is-deflate": "^1.0.0",
+    "is-gzip": "^1.0.0",
+    "is-stream": "^1.1.0",
+    "is-zip": "^1.0.0",
     "mime-db": "^1.33.0",
-    "pump": "^3.0.0"
+    "minipass": "2.3.5",
+    "peek-stream": "^1.1.0",
+    "pump": "^3.0.0",
+    "pumpify": "^1.3.3",
+    "string-to-stream": "^1.1.0",
+    "unzipper": "^0.8.9"
```

Comment:
Too many additional dependencies but I guess thats just the way it is in ecosystem :smile: